README.md

```diff
@@ -128,7 +122,7 @@ Alternatively, you can run this command, which will execute the printf, the echo
 <details>
 <summary>Dev Container</summary>
```

Comment:
We need to test the dev container to make sure it is still working for folks on Mac and Windows

Reply:
Do we know what layers we want to add to the devconainer

Reply:
I assume the same ones we had previously

Reply:
the devcontainer doesnt use isaac ros stuff right? Is there even a change to the dev container if all we changed were things that only run on the regular container?

Reply:
The devconainer is still built using the same build image script. We just don't include all layers because it causes image sizes to bloat.